client/src/api/v2/camera.js

```diff
@@ -39,3 +41,140 @@ export function useOverlays(device) {
 
   return { overlays, setActiveOverlay };
 }
+
+/**
+ * Send a message to server to start recording video
+ */
+export function startRecording() {
+  emit('start-camera-recording');
+}
+
+/**
+ * Send a message to server to stop recording video
+ */
+export function stopRecording() {
+  emit('stop-camera-recording');
+}
+
+/**
+ * Get the last MQTT payload for the status of a camera recording
+ *
+ * @param {string} device Camera
+ * @returns {string} Payload
+ */
+export function getLastPayload(device) {
```

Comment:
Not every function needs to be exported. In fact it's better to export as few as possible, as any exported functions become part of the public API and might require more refractors down the line if any breaking changes come along. I'd encourage you to only export any methods required by the display components, and keep any other helper methods within this API.

Reply:
Removed helper functions from public API interface.
4376700